### PR TITLE
Add periodically resetting utility meter config option

### PR DESCRIPTION
**Pull request by Wesley-Vos**
## Proposed change

The documentation is added to describe the new `periodically_resetting` option of the utility meter, proposed in the attached PR.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/88446

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by Wesley-Vos** [comment]
Hi! 

I see the bot message about the incorrect branch but I suppose that this is the correct branch. The change is needed for the linked PR, which is a new feature/ bugfix.

If I've to change something, please let me know!

File: source/_integrations/utility_meter.markdown [hunk 1]
```diff
@@ -55,6 +55,10 @@ Net consumption:
 Delta values:
   description: >
     Enable this if the source values are delta values since the last reading instead of absolute values. When this option is enabled, each new value received will be added as-is to the utility meter instead of adding the _difference_ between the new value and previous value.
+Periodically resetting:
+  description: >
+    Enable this if the source sensor state is expected to reset to 0, for example a smart plug that resets on boot.
+    When this options is disabled (for example if the source sensor is a domestic utility meter that never resets during the device's lifetime), the _difference_ between the new value and last valid value is added to the utility meter, which avoids the loss of a meter reading after the source sensor becomes available after being unavailable.
```
Comment (frenck) [inline source/_integrations/utility_meter.markdown]:
```suggestion
    Enable this if the source sensor state is expected to reset to 0, for example, a smart plug that resets on boot.
    When this option is disabled (for example, if the source sensor is a domestic utility meter that never resets during the device's lifetime), the _difference_ between the new value and the last valid value is added to the utility meter, which avoids the loss of a meter reading after the source sensor becomes available after being unavailable.
```

File: source/_integrations/utility_meter.markdown [hunk 2]
```diff
@@ -117,6 +121,11 @@ tariffs:
   required: false
   default: []
   type: list
+periodically_resetting:
+  description: Enable this if the source sensor state is expected to reset to 0, for example a smart plug that resets on boot. When this options is disabled (for example if the source sensor is a domestic utility meter that never resets during the device's lifetime), the _difference_ between the new value and last valid value is added to the utility meter, which avoids the loss of a meter reading after the source sensor becomes available after being unavailable.
```
Comment (frenck) [inline source/_integrations/utility_meter.markdown]:
```suggestion
  description: Enable this if the source sensor state is expected to reset to 0, for example, a smart plug that resets on boot. When this option is disabled (for example, if the source sensor is a domestic utility meter that never resets during the device's lifetime), the _difference_ between the new value and the last valid value is added to the utility meter, which avoids the loss of a meter reading after the source sensor becomes available after being unavailable.
```

**Review by frenck — APPROVED** [review]
Thanks, @Wesley-Vos 👍 

../Frenck

**Pull request merged**